libr/asm/arch/arm/aarch64/aarch64-dis.c

```diff
@@ -728,11 +728,15 @@ aarch64_ext_limm (const aarch64_operand *self ATTRIBUTE_UNUSED,
     {
       switch (S)
 	{
+#ifdef _MSC_VER
+#pragma message ("Windows: aarch64-dis.c aarch_ext_lim bypassed !")
```

Comment:
just change the switch to be an if (S => 0x00 && S <= 0x1f) { simd_size = 32 ...